constructr-coordination-consul/src/main/scala/com.tecsisa.constructr.coordination.consul/ConsulCoordination.scala

```diff
@@ -37,13 +39,15 @@ import akka.http.scaladsl.model.{
 }
 import akka.http.scaladsl.unmarshalling.Unmarshal
 import akka.stream.ActorMaterializer
-import akka.stream.scaladsl.{ Sink, Source }
+import akka.stream.scaladsl.{Sink, Source}
+
 import io.circe.Json
 import io.circe.parser.parse
+
 import de.heikoseeberger.constructr.coordination.Coordination
+
 import scala.concurrent.Future
-import scala.concurrent.duration.{ Duration, FiniteDuration }
-import scala.util.Try
+import scala.concurrent.duration.{Duration, FiniteDuration}
 
```

Comment:
Regarding the imports format, we plan to automate the `scalafmt` execution on every compilation. In the meantime, would you mind to execute `scalafmt` prior to push?. Thank!.